### PR TITLE
Sage Toast Initial

### PR DESCRIPTION
**Pull request by AndrwM**
## Description
- Css styles for Toast
  - Ability to stack multiple Toasts in the event many are generated at once
- Rails implementation of Toast
  - Rails implementation doesn't include JS – that will come in a wire-up PR later
- Docs: Update Element page header to match new markdown design

**Comp:** https://www.figma.com/file/SCZt9jMd7OcTMrNsOFifDp/Sage-Components-WIP?node-id=647%3A3223

### Screenshots

|  wide  |  mobile  |
|--------|--------|
|
image|image|

### Steps for testing
1. Check a11y
2. Check match of figma comps

## Related
BUILD-300

### CODE REVIEW

File: lib/sage-frontend/stylesheets/system/core/_functions.scss [hunk 1]
```diff
@@ -70,18 +70,18 @@
...
 /* ==================================================
 ** Replace `$search` with `$replace` in `$string`
   author Hugo Giraudel
   Usage: `str-replace(12px)`
-  
+
   Replace `$search` with `$replace` in `$string`
 
-  arguments: 
+  arguments:
```
Comment (AndrwM) [inline lib/sage-frontend/stylesheets/system/core/_functions.scss:84]: VS Code Trimmed the whitespace here

File: lib/sage-frontend/stylesheets/system/patterns/elements/_toast.scss [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  }
+}
+
+.sage-toast {
+  display: flex;
+  z-index: sage-z-index(modal, +1);
+  position: fixed;
+  justify-content: space-between;
+  align-items: center;
+  gap: sage-spacing();
```
Comment (philschanely) [inline lib/sage-frontend/stylesheets/system/patterns/elements/_toast.scss]: This isn't supported yet in safari with flex box :(

File: lib/sage-frontend/stylesheets/system/patterns/elements/_toast.scss [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  padding: 0 sage-spacing(md);
+  border-radius: sage-border(radius-large);
+  box-shadow: sage-shadow(lg);
+  background-color: sage-color(charcoal);
+  transition: bottom $sage-transition;
+  animation: 0.5s sage-toast--slide-in ease;
+
+  @mixin generate-stacked-toast-alignment($toast-height) {
```
Comment (philschanely) [inline lib/sage-frontend/stylesheets/system/patterns/elements/_toast.scss:36]: 😍

**Review by philschanely — APPROVED** [review]
Looking great Andrew. I've left a few notes. 🏆

File: lib/sage-frontend/stylesheets/system/patterns/elements/_toast.scss (hunk 2 above)
Comment (AndrwM) [reply lib/sage-frontend/stylesheets/system/patterns/elements/_toast.scss]: Ah, shoot, of course. Resolved

**Pull request merged**